horizon_bsn/bsndashboard/static/networktemplate/actions/create/createModal.html

```diff
@@ -0,0 +1,72 @@
+<div ng-form="createForm">
```

Comment:
probably have a TODO at the top of the file to update this with workflow when its available. 
reference the workflow feature pull request in upstream horizon.